influunt-app/app/styles/directives/influunt-tabs.css

```diff
@@ -86,3 +83,17 @@ li.ui-tabs-tab span.badge.badge-danger.badge-notification {
   left: 6px;
   border-radius: 0;
 }
+
+li.ui-state-active span.ui-icon.ui-icon-circle-close {
+  top: -34px;
+}
+
+.icheckbox_square-green {
+  float: left;
+}
+
+label span {
```

Comment:
Os arquivos com extensão `.css` são gerados a partir dos com extensão `.scss`. O que for adicionado no `.css` corre o risco de ser sobrescrito. Esse `label span` não está no `influunt-tabs.scss`, e pode ser que ele se perca. 

E outra coisa: esse seletor CSS está pegando todos os `span` dentro de uma `label`, ele está muito genérico. Não corre o risco de ele alterar o layout em outra parte do site que tenha um span dentro de uma label?